### DISCUSSION TITLE
Fantasy basketball - player daily scores / lineups

### DISCUSSION THREAD

**Discussion by Griljo** [cwendt94/espn-api#482] [ESPN API]
Hello,

Great app. I am still exploring it, but I can't find a way to pull the data about player daily scores (player, lineup slot, fantasy points score,..) from a points league. Anyone know if this is possilble?
Thanks.

**Comment by cwendt94** [comment]
You should be able to get all of that information using the League `box_scores` functionality which can be used to get current matchup information for a team and past data. See that section in the [wiki](https://github.com/cwendt94/espn-api/wiki/League-Class-Basketball) for more information.

**Reply by Griljo** [reply]
Thanks for help! I figured it out
